bumble/rfcomm.py

```diff
@@ -154,15 +152,54 @@ def make_service_sdp_records(
 
     if uuid:
         records.append(
-            ServiceAttribute(
-                SDP_SERVICE_CLASS_ID_LIST_ATTRIBUTE_ID,
-                DataElement.sequence([DataElement.uuid(uuid)]),
+            sdp.ServiceAttribute(
+                sdp.SDP_SERVICE_CLASS_ID_LIST_ATTRIBUTE_ID,
+                sdp.DataElement.sequence([sdp.DataElement.uuid(uuid)]),
             )
         )
 
     return records
 
 
+# -----------------------------------------------------------------------------
+async def search_channels_from_sdp(connection: Connection) -> Dict[int, List[UUID]]:
```

Comment:
There's a similar helper in `bench.py`. The other one doesn't return a list, but looks just for one specific UUID. I think both are useful. 
Maybe the single-uuid version from `bench.py` could be re-written by using this one, and moved to `rfcomm.py`.
We'd end up with:

- `rfcomm.find_rfcomm_channels` (this one, just different name) and 
- `rfcomm.find_rfcomm_channel_with_uuid` (the refactored one moved over from `bench.py`)

Reply:
I was wondering whether it should be a directional search or general search, but it seems not bad to have both.